torch/_inductor/fx_passes/quantization.py

```diff
@@ -562,11 +571,19 @@ def __init__(self, op_name: str, scalars_attr=None, algorithm_attr=None):
         # Priority 1 to match: QLinear Unary pattern with int8 output
         linear_unary_replace_patterns = {
             UnaryAttr("none", [], ""): generate_pattern_with_output_quant(
-                qlinear_pt2e_pattern,
+                get_qlinear,
                 dtype=original_pattern_output_dtype,
             ),
             UnaryAttr("relu", [], ""): generate_pattern_with_output_quant(
-                generate_pattern_with_unary(qlinear_pt2e_pattern, aten.relu.default),
+                generate_pattern_with_unary(get_qlinear, aten.relu.default),
+                dtype=original_pattern_output_dtype,
+            ),
+            UnaryAttr("gelu", [], "none"): generate_pattern_with_output_quant(
```

Comment:
As we synced offline, please follow up the int8-mixed-bf16 cases.

Reply:
create a new pull request to support int8-mixed-bf16 https://github.com/pytorch/pytorch/pull/116004